modules/markup/reStructuredText/reStructuredText_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package rst
+
+import (
+	"strings"
+	"testing"
+
+	"code.gitea.io/gitea/modules/setting"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const AppURL = "http://localhost:3000/"
+const Repo = "go-gitea/gitea"
+const AppSubURL = AppURL + Repo + "/"
```

Comment:
If you don't need all constants (look at comment for `defer`), merge them together. Also I think you can use unexported version. E.g. `const appSubURL = "http://localhost:3000/go-gitea/gitea/"`.